### DISCUSSION TITLE
Tuning & Reset-timer for ZG (and AQ20) as Catch-up raid and time-filler

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#3233] [Suggestions]
Traditional 40 player raids have been downsized to 25 players. Introducing 20 player raids on top will make the life for guild groups / raid leads a bit more complex.

Proposal:
- Instead of the official 72h reset timer for ZG, make the resets static on Tuesday -> Wednesday and Saturday -> Sunday.
- Allow up to 20 people in ZG, but tune it more in the sense of initial UBRS or for about 15 people. That way, people can do catch-up pugs with 20 people, but guild groups, who want to grind just 1-2 specific items, can clear ZG with 10-13 people (some guilds just split their 25 player roster into 2 groups for Onyxia already and could do so for ZG, too)

**Comment by Annamaria-CC** [comment]
Hey, thanks for your suggestion.

The tuning for ZG will be like Onyxia is, definitely not too hard, as it's a catchup raid. So the aim will be indeed that you can do it with less people if you want.

We cannot do your first suggestion, due to limitation of the core (we can only say how much time is between each reset, not an exact day of the week). As of now, we will be following the normal reset schedule.

**Comment by balleny** [comment]
Hm ok, but a weekly reset will also be really annoying. Most people will go ZG only for the enchants. For those, you need an item, dropping from 2 bosses only. If you consider that only 2 people will be able to get 1 enchant each per week, instead of 4 (considering 72h reset), it will have a negative influence.

**Reply by Nyeriah** [reply]
I believe the normal reset schedule in this case refers to 72 hours (3 days) as that's the reset currently set for ZG.

**Reply by Annamaria-CC** [reply]
Correct 72h

**Reply by balleny** [reply]
alright, makes sense - considering the change to Onyxias reset schedule, I was thinking the same might take place here.
